demo/x-local-translate.html

```diff
@@ -45,12 +45,6 @@
 
       properties: {
         /* Overriden from AppLocalizeBehavior */
-        language: {
-          value: 'en',
```

Comment:
This was already setting a default value of `en` on this element -- I'm not sure I understand why this didn't work for you. You could also do 
```
language: {
  value: Polymer.defaultLanguage
}
```
here in your app if you wanted (but note, not in the behaviour itself like you've done it)